inspector/scripts/requestAndRead.ts

```diff
@@ -0,0 +1,105 @@
+import { deployments, ethers, getNamedAccounts } from "hardhat";
+import {
+  getKeyHash,
+  estimateVRFServiceFee,
+  estimateRRServiceFee,
+  getAggregatorPairs,
+  priceFeeds,
+  findPairsWithSameRoundId,
+} from "./utils";
+
+async function main() {
+  const {
+    sResponse: rrResponseBefore,
+    sRandomWord: vrfResponseBefore,
+    priceFeedResults: priceFeedsBefore,
+  } = await read();
+
+  await request();
+
+  // wait 30 seconds for fulfillment and submission
+  await new Promise((resolve) => setTimeout(resolve, 30000));
+  const {
+    sResponse: rrResponseAfter,
+    sRandomWord: vrfResponseAfter,
+    priceFeedResults: priceFeedsAfter,
+  } = await read();
+
+  if (rrResponseBefore == rrResponseAfter) {
+    throw "check if request response is alive";
+  }
+  if (vrfResponseBefore == vrfResponseAfter) {
+    throw "check if vrf is alive";
+  }
+  const result = findPairsWithSameRoundId(priceFeedsBefore, priceFeedsAfter);
+  if (result.length > 0) {
+    throw `check following pairs:${result}`;
+  }
+
+  console.log("successful");
+}
+
+async function read() {
+  const _inspectorConsumer = await deployments.get("InspectorConsumer");
+  const inspectorConsumer = await ethers.getContractAt(
+    _inspectorConsumer.abi,
+    _inspectorConsumer.address
+  );
+
+  const sResponse = await inspectorConsumer.sResponse();
+  const sRandomWord = await inspectorConsumer.sResponse();
+
+  const priceFeedResults: priceFeeds = {};
+  const pairs = await getAggregatorPairs();
+  for (const pair of pairs) {
+    const priceFeedResult = await inspectorConsumer.requestDataFeed(pair);
+    priceFeedResults[pair] = {
+      roundId: priceFeedResult[0],
```

Comment:
I think that `roundId` and `answer` are not that important since this is meant to check whether data feeds are being updated. I would recommend using `startedAt` instead and check if it data is too old.

Reply:
I'll remove datafeed check up functionalities, it'll be later checked through graph-node reference for submit event

Reply:
Okay!